### PR TITLE
Weak password reset implementation

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/sensitive_data_exposure/weak_password_reset_implementation/template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Weak Password Reset Implementation
+
+## Overview of the Vulnerability
+
+When the password reset implementation is weak, the strength of the overall authentication process for the application is diminished. Tokens sent over HTTP, predictable reset tokens, and long expiry times create weak conditions for the password reset implementation. This application’s weak password reset implementation allows an attacker to intercept the password reset token and reset a user’s password, locking the user out of their account and achieving full account takeover.
```
Comment (evildaemond) [inline submissions/description/sensitive_data_exposure/weak_password_reset_implementation/template.md:5]: The wording used here doesn't convey what the vulnerability is, in this case you've specified this to interception, there are other methods which could be used, such as predictable tokens. I've modified it below to be more generic here. 

> This application’s weak password reset implementation allows an attacker to abuse the password reset token and reset a user’s password, locking the user out of their account and achieving full account takeover.

**Review by evildaemond — CHANGES_REQUESTED** [review]
1 change then LGTM

File: submissions/description/sensitive_data_exposure/weak_password_reset_implementation/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/sensitive_data_exposure/weak_password_reset_implementation/template.md:5]: Thanks, great pickup.

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**